migrations/Version202403071639280420_qtiItemPci.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use Doctrine\Migrations\Exception\IrreversibleMigration;
+use oat\qtiItemPci\model\IMSPciModel;
+use oat\qtiItemPci\scripts\install\RegisterPciLikertScaleInteraction;
+
+/**
+ * phpcs:disable Squiz.Classes.ValidClassName
+ */
+final class Version202403071639280420_qtiItemPci extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Install/update PCI LikertScaleInteraction';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $registry = (new IMSPciModel())->getRegistry();
```

Comment:
**issue:** This is not an IMS PCI...
**suggestion:**
```suggestion
        $registry = (new PciModel())->getRegistry();
```

Reply:
> This is not an IMS PCI... Did you create it using the SDK?

Guilty.